test/cypress/integration/group-creation.spec.js

```diff
@@ -0,0 +1,91 @@
+describe('Group Creation and Inviting Members', () => {
+  const userId = new Date().getMilliseconds()
+
+  it('successfully loads the homepage', function () {
+    cy.visit('/')
+  })
+
+  it('register user1 and logout', () => {
+    cy.giSignUp(`user1-${userId}`)
+    cy.giLogOut()
+  })
+
+  it('register user2', () => {
+    cy.giSignUp(`user2-${userId}`)
+  })
```

Comment:
Missing several created users from old tests

Reply:
For this scenario, of creating a group and inviting one member, there's no need to create multiple users.

When we write the tests for the Proposals, I'll do that part.